From dbc791c8f24ffac0d98e86213e4d592660f6087c Mon Sep 17 00:00:00 2001
From: Michel Thebeau <michel.thebeau@windriver.com>
Date: Fri, 19 Aug 2016 09:21:44 -0400
Subject: [PATCH] CGTS-4685: setup: remove unused default groups

A customer request to remove unused users and groups.  These are default
users/groups on centos which have no bearing on a Titanium Cloud cluster.

Two additional groups are removed: mem, man

Signed-off-by: Michel Thebeau <michel.thebeau@windriver.com>
---
 group | 2 --
 1 file changed, 2 deletions(-)

diff --git a/group b/group
index 42a8ed1..a3bb69e 100644
--- a/group
+++ b/group
@@ -2,12 +2,10 @@ root::0:
 sys::3:
 tty::5:
 disk::6:
-mem::8:
 kmem::9:
 wheel::10:
 cdrom::11:
 mail::12:
-man::15:
 dialout::18:
 floppy::19:
 tape::33:
-- 
2.7.4

